.github/workflows/Build.yml

```diff
@@ -33,6 +33,11 @@ jobs:
           pip install flake8 pytest pytest-cov
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
           if [ -f tests_requirements.txt ]; then pip install -r tests_requirements.txt; fi
+      - name: Run pre-commit hooks
+        env:
+          LEAKTK_PATTERN_SERVER_AUTH_TOKEN: ${{ secrets.LEAKTK_PATTERN_SERVER_AUTH_TOKEN }}
```

Comment:
Why we need to pass token LEAKTK_PATTERN_SERVER_AUTH_TOKEN ?

Reply:
rh-gitleaks token